package.json

```diff
@@ -75,7 +75,8 @@
     "yaml": "^2.2.2",
     "semver": "^7.5.2",
     "optionator": "~0.9.3",
-    "typescript": "~5.1.6"
+    "typescript": "~5.1.6",
+    "@cypress/request": "^3.0.0"
```

Comment:
Where is `@cypress/request` needed/used? I could not find it with the VSCode keyword search.

Is it used as a dependency of some other package and not in our code? If so, by explicitly stating a version here, are you forcing to use the latest version? Or should we use yarn's [`resolutions`](https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/#toc-how-to-use-it)?

Reply:
you can do a find in the lockfile or by running `npm ls @cypress/request`.

and yes i am forcing the latest version using `resolutions`, because `^2.88.11` is not compatible with `3.0.0`